### DISCUSSION TITLE
Why not implement FHSS for SX127x?

### DISCUSSION THREAD

**Discussion by xg590** [sandeepmistry/arduino-LoRa#517] [General]
I just implemented the frequency hopping for SX1276. Our of curiosity, has any library implemented FHSS before? Why not?

It seems FHSS is a must if I want to send a long packet with large SF and narrow BW. Otherwise, a wrong combination can easily exceeds FCC's 400ms dwell time threshold.

**Comment by morganrallen** [comment]
No one has implemented and so far I don't think anyone else has asked for it.
